APIMatic.Core/Response/ResponseHandler.cs

```diff
@@ -138,6 +138,16 @@ internal ReturnType Result<ReturnType>(CoreContext<CoreRequest, CoreResponse> co
             throw new InvalidOperationException($"Unable to transform {typeof(ResponseType)} into {typeof(ReturnType)}. ReturnTypeCreator is not provided.");
         }
 
+        private bool HasEmptyResponse(CoreResponse response)
+        {
+            var resType = typeof(ResponseType);
+            if (resType == typeof(VoidType))
+            {
+                return true;
+            }
+            return string.Equals(response.Body, string.Empty) && CoreHelper.IsNullableType(resType);
```

Comment:
Blocker: As we agreed upon checking the whitespaces in the body as well therefore the response.Body should be trimmed before checking the equality here.